stock_picking_partner_note/views/stock_picking_note.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_stock_picking_note_form" model="ir.ui.view">
+        <field name="name">stock.picking.note.form</field>
+        <field name="model">stock.picking.note</field>
+        <field name="arch" type="xml">
+            <form string="Picking Note">
+                <group>
+                    <field name="name" />
+                    <field name="note_type_id" />
+                    <field name="active" />
+                </group>
```

Comment:
Would be useful to see the list of partners having this note

Reply:
ok, but use a smart btn rather than a direct listing of m2m

Reply:
nooooo :fearful: 
Keop it simple, just list them, there is nothing on that view. Easier to visualize and change

Reply:
A smart button wouldn't work because `stock_picking_note_ids` because is a M2M and not O2M.
With a button you would be able to create a new note BUT detached from the partner.
I keep the simple version showing the notes directly in the form

Reply:
> A smart button wouldn't work because stock_picking_note_ids because is a M2M and not O2M.
> With a button you would be able to create a new note BUT detached from the partner.

This is wrong. You will have an action to handle the btn and you can search for the records you need. No matter if o2m or m2m. 
Regarding creation, that's wrong too: you simply have to set the default partner in the context.

> I keep the simple version showing the notes directly in the form

Ok, but that's about time and not about possibilities :wink: